src/go/k8s/config/crd/bases/redpanda.vectorized.io_clusters.yaml

```diff
@@ -1184,6 +1184,18 @@ spec:
                               description: IP is set for load-balancer ingress points
                                 that are IP based (typically GCE or OpenStack load-balancers)
                               type: string
+                            ipMode:
```

Comment:
Is this change coming from the K8s API upgrade? I don't see an "optional" field set on this which is a bit concerning to me. Will there be any trouble with existing installations?

Reply:
I think this is because someone forgot to run make generate earlier.